samples/deployment/fabric-smart-client/the-simple-testing-network/Makefile

```diff
@@ -1,12 +1,18 @@
 # Copyright IBM Corp. All Rights Reserved.
 #
 # SPDX-License-Identifier: Apache-2.0
+TOP = ../../../..
+include $(TOP)/build.mk
 
 FABRIC_LOGGING_SPEC?="fpc=debug:endorser,grpc,comm.grpc,gossip=error:info"
 
 build:
 	go build -o tstn
 
+images: 
+	$(DOCKER) pull hyperledger/fabric-ccenv:2.5
+	$(DOCKER) image tag hyperledger/fabric-ccenv:2.5 hyperledger/fabric-ccenv:latest
+
```

Comment:
should we also include the hyperledger explorer images here?

Reply:
I thought of that. I would have to change the Readme a bit more. I was going for the least modif, but I could add the explorer images and merge that part ....

Reply:
just made the changes :-)